lib/components-schema-v1_11_x.ts

```diff
@@ -101,6 +101,10 @@ const componentGroupDefinition: JSONSchema7Definition = {
                 minLength: 3,
             },
             label: labelProperty('Group label shown in Digital Editor'),
+            integrationLogo: {
```

Comment:
Here we use it for an integration but this could be a group logo for other groups too, don't you agree?

Reply:
I think it is fine for now to have just bottom right. The setup allows us to easily extend it if needed.